### PR TITLE
Extend TYPO3 Exception 1381512761 for -1 VersionState

### PR DESCRIPTION
**Pull request by DanielSiepmann**
Relates: https://forge.typo3.org/issues/99401

### CODE REVIEW

**Review by linawolf — APPROVED** [review]

File: Documentation/Exceptions/1381512761.rst [hunk 1]
```diff
@@ -9,6 +9,13 @@ TYPO3 Exception 1381512761
 Invalid value "..." for enumeration "..."
 =========================================
 
+Invalid value "-1" for enumeration "TYPO3\CMS\Core\Versioning\VersionState"
```
Comment (brotkrueml) [inline Documentation/Exceptions/1381512761.rst:12]: Should we add the version number (v11+) somewhere?

Comment (DanielSiepmann) [reply Documentation/Exceptions/1381512761.rst:12]: That's up to you. But I guess it is 12+ as v11 only deprecated the -1.
I thought it is enough to have the message of the Exception.

**Review by lolli42 — APPROVED** [review]

File: Documentation/Exceptions/1381512761.rst [hunk 2]
```diff
@@ -9,6 +9,13 @@ TYPO3 Exception 1381512761
 Invalid value "..." for enumeration "..."
```
Comment (brotkrueml) [inline Documentation/Exceptions/1381512761.rst:9]:
```suggestion
Invalid value "..." for enumeration "..." in v12
```

File: Documentation/Exceptions/1381512761.rst [hunk 3]
```diff
@@ -9,6 +9,13 @@ TYPO3 Exception 1381512761
...
 =========================================
```
Comment (brotkrueml) [inline Documentation/Exceptions/1381512761.rst:10]:
```suggestion
================================================
```

**Pull request merged**